features/record-detail/components/swim-stats-item.tsx

```diff
@@ -0,0 +1,39 @@
+import { css } from '@/styled-system/css';
+import { flex } from '@/styled-system/patterns';
+
+export const SwimStatsItem = () => {
+  return (
+    <div className={containerStyle}>
+      <h1 className={text.type}>자유형</h1>
+      <h2 className={text.distance}>1500m</h2>
+      <p className={text.lap}>124lap</p>
+    </div>
+  );
+};
+
+const containerStyle = flex({
+  gap: '2px',
+  flexDirection: 'column',
+  w: 'full',
+});
+
+// TODO: swim type에 따른 색상 변경
+const text = {
+  type: css({
+    textStyle: 'label1.normal',
+    fontWeight: 'medium',
+    color: 'primary.swim.자유형.default',
+  }),
+
+  distance: css({
+    textStyle: 'label1.normal',
+    fontWeight: 'bold',
+    color: 'text.neutral',
+  }),
+
+  lap: css({
+    textStyle: 'label1.normal',
+    fontWeight: 'medium',
+    color: 'text.placeHolder',
+  }),
+};
```

Comment:
미터와 바퀴수 모두 "거리" 와 관련되어 있고 단위만 다른 것이고 / label1.normal 텍스트 스타일도 겹치기 때문에 마찬가지로 cva 활용해주고, 거리 "단위"로 분기처리 해주면 좀 더 명확해질 것 같다는 의견 드립니다.

```suggestion
distance: cva({
    base: {
      textStyle: 'label1.normal',
    },
    variants: {
      unit: {
        meter: { fontWeight: 'bold', color: 'text.neutral' },
        laps: { fontWeight: 'medium', color: 'text.placeHolder' },
      },
    },
  }),